l10n_eu_oss_oca/README.rst

```diff
@@ -7,7 +7,7 @@ L10n EU OSS OCA
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-   !! source digest: sha256:fde85cb88c420131b91b360fbb84f553d4977cbbfc80cc868c52291833e08e03
+   !! source digest: sha256:9f24da0b57196401086a0a54b7bb7e24024c260cc4c90d42308810512634a55c
```

Comment:
Why this change and the one in the index.html?

Reply:
Were residual changes from the first pre-commit when I added a new migration script